npc/re/jobs/novice/academy.txt

```diff
@@ -13,75 +13,241 @@
 //= 1.0.0 Initial Release. [Kisuka]
 //= 1.0.1 Converted item constants to IDs and added hair coupon. [Kisuka]
 //= 2.0.0 Converted from Hercules Script to rAthena Script. [Capuche]
+//= 2.0.1 Add new Intro Renewal based on Hercules [Jenkijo]
 //==============================================================================
 
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
```

Comment:
Ok glad to say that is not stored in the database. But the OnTouch area is bigger. If you leave through the portal on the right, relog and then go back to the left you instantly trigger the navigation again. You get spawned to 22,30 so it at least needs to reach to there.